pettingzoo/__init__.py

```diff
@@ -1,7 +1,7 @@
 import os
 import sys
 
-from pettingzoo.utils import AECEnv, ParallelEnv
```

Comment:
you can remove this change, we purposefully expose AECEnv and ParallelEnv in the utils init file to make it easier (people just need to know they're int utils not the specific file they're in)